CONTRIBUTING.md

```diff
@@ -37,4 +40,8 @@ While this software is fully open source and not affiliated with HoYoverse, Cogn
 - [Genshin Impact Terms of Service](https://genshin.hoyoverse.com/en/company/terms)
 - [Honkai Impact 3rd Terms of Service](https://honkaiimpact3.hoyoverse.com/global/en-us/terms)
 - [Honkai: Star Rail Terms of Service](https://hsr.hoyoverse.com/en-us/company/terms)
-- [HoYoLAB Forum Terms of Service](https://www.hoyolab.com/agreement)
\ No newline at end of file
+- [HoYoLAB Forum Terms of Service](https://www.hoyolab.com/agreement)
+
+# A Humble Thank You
+We as contributors always feel grateful for all your contribution to the project, whether its a Language/Localization help, helping us add more features, bug fixes/reports, and even for using this launcher. With all your help we all can make this project alive and well in the hope of making a good launcher that can help brings more Quality of Life improvements over official launcher.

```

Comment:
- "As contributors, we [...]" has a better ring to it and is the grammatically correct way :P 
- "whether its a Language [...]" -> "whether it be through helping with localization, helping us come up with new features, reporting bugs and using this launcher", again, typos but the overall sentence is ok
- Rephrase the last sentence: "Through everyone's effort, we can keep this project alive by bringing even more features and quality-of-life (QoL) upgrades over the existing (and official) launchers out there.

Reply:
Honestly, I don't know why I had so many mistakes just trying to type this paragraph LMAO
Fixed in https://github.com/neon-nyan/Collapse/pull/120/commits/88402163a7f842b2ba659213bb2c404783b2d2c1

Reply:
You made a typo:

- "whether it be though" -> "whether it be through"

XD

Reply:
oh. my. gawd. my eyes

fixed